modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/model/listener/test/CompanyModelListenerTest.java

```diff
@@ -43,8 +43,11 @@ public class CompanyModelListenerTest {
 
 	@Test
 	public void testSXPElements() throws Exception {
-		List<SXPElement> sxpElements = ReflectionTestUtil.getFieldValue(
-			_modelListener, "_sxpElements");
+		Object companyModelListenerHelper = ReflectionTestUtil.getFieldValue(
```

Comment:
@MrJohn1911 This test only tests the `SXPElements` related thing in `CompanyModelListener`. When you extract these logic out, the test should be also altered to test that new class directly.